rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -619,4 +619,19 @@ public static int calculatePegoutTxSize(ActivationConfig.ForBlock activations, F
 
         return baseSize + signingSize;
     }
+
+    public static Script getFederationMembersP2SHScript(ActivationConfig.ForBlock activations, Federation federation) {
+        // when the federation is a standard multisig,
+        // the members p2sh script is the p2sh script
```

Comment:
nit: I think there's space for this to be in a single line with the above. It's a bit confusing I think.